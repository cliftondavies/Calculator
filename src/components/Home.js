const Home = () => {
  const paragraph = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Consequat nisl vel pretium lectus quam id leo in. Suspendisse in est ante in nibh mauris cursus mattis molestie. Arcu felis bibendum ut tristique et egestas quis. Suspendisse in est ante in. Tortor consequat id porta nibh venenatis cras sed. Blandit turpis cursus in hac habitasse platea dictumst quisque. Est lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque. Ac feugiat sed lectus vestibulum. Tellus elementum sagittis vitae et leo duis ut diam quam. Dui sapien eget mi proin sed libero enim sed faucibus. Magna sit amet purus gravida. Et ultrices neque ornare aenean euismod elementum nisi quis eleifend. Consequat interdum varius sit amet mattis vulputate.';

  return (
    <main className="home">
      <h2>
        Welcome to our page!
      </h2>

      <p>
        {paragraph}
      </p>

      <p>
        {paragraph}
      </p>
    </main>
  );
};

export default Home;
