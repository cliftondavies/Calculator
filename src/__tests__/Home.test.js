import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('renders elements on home page', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
