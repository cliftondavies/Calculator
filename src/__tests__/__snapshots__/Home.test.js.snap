// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements on home page 1`] = `
<h2>
  Home
</h2>
`;
